plugins/modules/elastic_endpoint_security.py

```diff
@@ -0,0 +1,172 @@
+          i=0
+          for input in pkg_policy_object['inputs']:
+            if input['type'] == 'endpoint':
+                ########## Updating configuration
+                pkg_policy_object['inputs'][i]['config']['policy']['value']['windows']['antivirus_registration']['enabled'] = self.endpoint_security_antivirus
+                ########## Removing values to reapply the JSON with the above values changed
+                pkg_policy_object_id = pkg_policy_object['id'] 
+                pkg_policy_object.pop('id')
+                pkg_policy_object.pop('revision')
+                pkg_policy_object.pop('created_at')
+                pkg_policy_object.pop('created_by')
+                pkg_policy_object.pop('updated_at')
+                pkg_policy_object.pop('updated_by')
+                break
+            i=+1
+          pkg_policy_object_json = json.dumps(pkg_policy_object)
+          pkg_policy_update = kibana.update_pkg_policy(pkg_policy_object_id,pkg_policy_object_json)
+          results['pkg_policy_update_status'] = "Updating Endpoint Security Package"
+          pkg_policy_info = pkg_policy_update
+        
+        elif pkg_policy_object['package']['title'] == 'Prebuilt Security Detection Rules' and self.prebuilt_rules_activate == True and self.module.check_mode == False:
+              #SecurityRules = rules_action.activating_all_rules(self,50)
+              pkg_policy_info = kibana.activate_rule(50,'Endpoint Security')
+        
+        elif self.module.check_mode == True:
+          results['pkg_policy_update_status'] = "Check mode is set to True, not going to update pkg policy"
+        
+        return pkg_policy_info
+
+def main():
+
+    module_args=dict(   
+        host=dict(type='str',Required=True),
+        port=dict(type='int', default=9243),
+        username=dict(type='str', Required=True),
+        password=dict(type='str', no_log=True, Required=True),   
+        verify_ssl_cert=dict(type='bool', default=True),
+        agent_policy_id=dict(type='str'),
+        agent_policy_name=dict(type='str'),
+        integration_name=dict(type='str', Required=True),
+        pkg_policy_name=dict(type='str', Required=True),
+        pkg_policy_desc=dict(type='str'),
+        endpoint_security_antivirus=dict(type='bool', default=True),
+        prebuilt_rules_activate=dict(type='bool', default=True),
+        state=dict(type='str', default='present')
+    )
+    argument_dependencies = []
+        #('state', 'present', ('enabled', 'alert_type', 'conditions', 'actions')),
+        #('alert-type', 'metrics_threshold', ('conditions'))
+    
+    module = AnsibleModule(argument_spec=module_args, required_if=argument_dependencies, supports_check_mode=True)
+    state = module.params.get('state')
+    
+    if module.check_mode:
+        results['changed'] = False
+    else:
+        results['changed'] = True
+        
+    kibana = Kibana(module)
```

Comment:
In order to get the params defined in `SecurityBaseline` `init`, `Kibana` should be replaced here with `SecurityBaseline`